app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreen.kt

```diff
@@ -413,9 +368,8 @@ private fun ContentFooter(
                         onOpenConversation,
                         onCancelConnectionRequest,
                         acceptConnectionRequest,
-                        ignoreConnectionRequest,
-                        onUnblockUser
-                    )
+                        ignoreConnectionRequest
+                    ) { onUnblockUser(state.userName) }
```

Comment:
Why username and not user Id?

Reply:
`userId` is needed for unblocking the user - yes, but it's a `val` in ViewModel, not need duplicate it here
And `username` is needed to display a correct error, if it occurs